pkg/virtctl/portforward/portforward.go

```diff
@@ -0,0 +1,220 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package portforward
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+const (
+	forwardToStdoutFlag = "stdio"
+	addressFlag         = "address"
+)
+
+var (
+	forwardToStdout bool
+	address         string = "127.0.0.1"
+)
+
+func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "port-forward [kind/]name[.namespace] [protocol/]localPort[:targetPort]...",
+		Short:   "Forward local ports to a virtualmachine or virtualmachineinstance.",
+		Long:    usage(),
+		Example: examples(),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if n := len(args); n < 2 {
+				fmt.Printf("fatal: Number of input parameters is incorrect, portforward requires at least 2 arg(s), received %d\n\n", n)
```

Comment:
Is `cmd.Help()` also printed to stdout or does it go to stderr? Let us just ensure that we print to the same channel.

Reply:
Printing it to stderr now by default and made sure cobra also prints to the right output when using stdio.